.gitmodules

```diff
@@ -1051,9 +1051,9 @@
 	path = vendor/grammars/vscode-codeql
 	url = https://github.com/github/vscode-codeql
 	branch = main
-[submodule "vendor/grammars/vscode-cue"]
-	path = vendor/grammars/vscode-cue
-	url = https://github.com/cue-sh/vscode-cue
```

Comment:
Whoops. The CI failure is happening because you're accidentally removing the Cue grammar.

Reply:
Really? Ooops! Stand by...

Reply:
Fixed.